database/insertData/dev/04_users.js

```diff
@@ -488,6 +483,57 @@ exports.queries = [
       }
     }
   }`,
+  // Final decision users
+  `mutation {
+    createUser(
+      input: {
+        user: {
+          username: "finalDecision1"
+          firstName: "Final"
+          lastName: "Decision Maker 1"
+          passwordHash: "$2a$10$5R5ruFOLgrjOox5oH0I67.Rez7qGCEwf2a60Pe2TpfmIN99Dr0uW."
```

Comment:
Create new hashes for these guys, if you can be bothered: https://www.browserling.com/tools/bcrypt 

Not really important for "dev" users, I know, so feel free to ignore.

Reply:
Thanks Done